packages/mdc-button/_mixins.scss

```diff
@@ -243,6 +243,25 @@ $mdc-button-ripple-target: ".mdc-button__ripple";
   );
 
   @include mdc-button-height($height, $query: $query);
+
+  @if $density-scale != 0 {
+    @include mdc-button-touch-target-reset_;
+  }
+}
+
+///
+/// Resets touch target-related styles. This is called from the density mixin to
+/// automatically remove the increased touch target, since dense components
+/// don't have the same default a11y requirements.
```

Comment:
Reminder to update this comment to reflect internal edits.

Reply:
This was updated :)